numba/cuda/tests/cudapy/test_atomics.py

```diff
@@ -20,229 +20,154 @@ def skip_unless_cc_32(fn):
 def skip_unless_cc_50(fn):
     return unittest.skipUnless(cc_X_or_above(5, 0), "require cc >= 5.0")(fn)
 
+@cuda.jit(device=True)
+def atomic_cast_to_uint64(num):
+    return uint64(num)
 
-def atomic_add(ary):
+
+@cuda.jit(device=True)
+def atomic_cast_to_int(num):
+    return int(num)
+
+
+@cuda.jit(device=True)
+def atomic_cast_none(num):
+    return num
+
+
+@cuda.jit(device=True)
+def atomic_binary_1dim_shared(ary, idx, op2, ary_dtype, ary_nelements, binop_func, cast_func, initializer):
```

Comment:
long line, for new code we try and keep to <80 chars https://numba.readthedocs.io/en/stable/developer/contributing.html#coding-conventions